src/lib/selection/media/html-selector.js

```diff
@@ -373,12 +400,18 @@ export default class HTMLSelector extends MediaSelector {
     textSelector.context = contextStr
     textSelector.position = contextPos
 
+    textSelector.checkContext = checkContext
+
     const prefix = selection.anchorNode.data.substr(0, textSelector.start).trim().replace(/\n/g, '')
     const suffix = selection.anchorNode.data.substr(textSelector.end).trim().replace(/\n/g, '')
     textSelector.createTextQuoteSelector(prefix, suffix)
     return textSelector
   }
 
+  defineCheckContextFromContext (contextStr) {
+    return contextStr.indexOf(' ') === -1 ? contextStr : contextStr.substr(0, contextStr.indexOf(' '))
+  }
+
```

Comment:
checkContextForward is defined as all contextForward before space or punctuation